### PR TITLE
[DE] Add scene and script

### PR DESCRIPTION
**Pull request by steffenrapp**

### CODE REVIEW

File: tests/de/scene_HassTurnOn.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: de
+tests:
+  - sentences:
```
Comment (mib1185) [inline tests/de/scene_HassTurnOn.yaml:3]: Please add test sets for each possible variant

File: tests/de/script_HassTurnOn.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+language: de
+tests:
+  - sentences:
```
Comment (mib1185) [inline tests/de/script_HassTurnOn.yaml:3]: Please add test sets for each possible variant

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @steffenrapp 
thanks again for contributing 🙂 
Sure, there are some comments from my side 🙈

File: sentences/de/scene_HassTurnOn.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+language: de
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "<aktivieren> [<szene>|die] <name> [<szene>]"
+          - "[<szene>|die] <name> [<szene>] (<aktivieren>|<an>)"
```
[the remark this answers is not in the document]
Comment (steffenrapp) [reply sentences/de/scene_HassTurnOn.yaml:7]: Alright, I'll split it up but the "die" was there to make sentences like "aktiviere die Filmabend Szene" possible, so I have to add it sometimes.

**Comment by steffenrapp** [comment]
> Hi @steffenrapp 
> 
> thanks again for contributing 🙂 
> 
> Sure, there are some comments from my side 🙈 

Hi, thank you. Hope I got them all 😄

File: tests/de/scene_HassTurnOn.yaml (hunk 1 above)
Comment (mib1185) [reply tests/de/scene_HassTurnOn.yaml:3]: i count 15 possible variants, but only 7 test sentences 😁

File: tests/de/script_HassTurnOn.yaml (hunk 2 above)
Comment (mib1185) [reply tests/de/script_HassTurnOn.yaml:3]: i count 12 possible variants, but only 7 test sentences 😁

**Review by mib1185 — CHANGES_REQUESTED** [review]
we still need more test sentences 🙂

File: tests/de/scene_HassTurnOn.yaml (hunk 1 above)
Comment (steffenrapp) [reply tests/de/scene_HassTurnOn.yaml:3]: You mean for every combination which each of the expansion rules? Shouldn't it be fine to have tests for each line of the defined sentences?

Comment (mib1185) [reply tests/de/scene_HassTurnOn.yaml:3]: We should have tests for every possible variant, so we can ensure that later changes do not brake already working sentences (_avoid regressions_)

Comment (steffenrapp) [reply tests/de/scene_HassTurnOn.yaml:3]: Alright, makes sense. Found some more 😄

**Review by mib1185 — APPROVED** [review]
Thx @steffenrapp 👍

**Pull request merged**